components/ws-manager-bridge/ee/src/prebuild-updater-db.ts

```diff
@@ -79,6 +80,10 @@ export class PrebuildUpdaterDB implements PrebuildUpdater {
                 span.setTag("updatePrebuildWorkspace.prebuild.state", updatedPrebuild.state);
                 span.setTag("updatePrebuildWorkspace.prebuild.error", updatedPrebuild.error);
 
+                if (updatedPrebuild.state && terminatingStates.includes(updatedPrebuild.state)) {
```

Comment:
The `writeToDB`/`is_governing` label should not be in the metric but checked in this if statement:
`if (writeToDB && terminatingStates.includes(updatedPrebuild.state)) { .... }`